app/styles/ui/window/_tooltips.scss

```diff
@@ -160,4 +160,12 @@ body > .tooltip,
       margin-right: var(--spacing);
     }
   }
+
+  .title {
```

Comment:
Once we move the tooltip out of the autocompleting text input we can give it a specific class name like `overflow-hint` or something and move these styles inside of that like we do with `user-info` above.